packages/ui/package.json

```diff
@@ -55,6 +55,10 @@
     "./ErrorMessage": {
       "types": "./dist/alert/ErrorMessage.svelte.d.ts",
       "svelte": "./dist/alert/ErrorMessage.svelte"
+    },
+    "./SearchInput": {
+      "types": "./dist/input/SearchInput.svelte.d.ts",
+      "svelte": "./dist/input/SearchInput.svelte"
```

Comment:
we might want to rename to `inputs` since we already have other items in it, make sense to use plural